daemon/daemon.go

```diff
@@ -1347,14 +1345,6 @@ func (daemon *Daemon) setGenericResources(conf *config.Config) error {
 	return nil
 }
 
-func setDefaultMtu(conf *config.Config) {
-	// do nothing if the config does not have the default 0 value.
-	if conf.Mtu != 0 {
-		return
-	}
-	conf.Mtu = config.DefaultNetworkMtu
```

Comment:
Does this technically mean we can no longer set `--mtu=0` or `"mtu":0` in our configuration to receive the default value? :innocent:

(Not sure whether anyone's actually doing that, but the current code would support that, right?  The "workaround" for anyone doing that would just be to remove the flag/value, right?  Unless the flag can override the configuration or something but I think that's an error so this is probably fine.)